crates/sui-core/src/generate_format.rs

```diff
@@ -97,7 +92,7 @@ fn get_registry() -> Result<Registry> {
     tracer.trace_type::<Argument>(&samples)?;
     tracer.trace_type::<Command>(&samples)?;
     tracer.trace_type::<CommandArgumentError>(&samples)?;
-    tracer.trace_type::<PackageUpgradeError>(&samples)?;
```

Comment:
Bad merge?

Reply:
No, it's a sub enum that was unused exist